MastodonSDK/Sources/MastodonUI/View/Content/StatusView+Configuration.swift

```diff
@@ -130,6 +130,7 @@ extension StatusView {
                                     authorization: authenticationBox.userAuthorization
                                 ).singleOutput()
                             }
+                            .receive(on: DispatchQueue.main)
                             .sink { completion in
                                 // do nothing
                             } receiveValue: { [weak self] response in
```

Comment:
I think this can be removed so it's just `.sink { [weak self] response in`

Reply:
Thank you.

The Combine requires handling the `completion` case. Due to this event pipeline could emit an `Error`. So I cannot remove that.

Reply:
Ah, that makes sense!